src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -172,6 +172,9 @@
         - name: --vnet-subnet-id
           type: string
           short-summary: The ID of a subnet in an existing VNet into which to deploy the cluster.
+        - name: --ppg
```

Comment:
--ppg-id?

Reply:
According to cli command guidelines https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#argument-naming-conventions 
ID Arguments:
Arguments that expect ARM IDs should omit -id but call out that an ARM ID is allowed in the help text. These are often used as part of the "name or ID" convention (i.e: --storage-account can often accept a storage account name OR ARM ID).